nmostesting/Config.py

```diff
@@ -137,6 +137,20 @@
     "video_exactframerate": "25"
 }
 
+# IS-11 Media Profiles parameters which can be modified for devices which do not support the defaults
+# IS-11 Media Profiles testing is not concerned with support for specific media parameters, but must include them in the file
+MEDIA_PROFILES_PREFERENCES = {
```

Comment:
Are there valid situations where the these values would differ from SDP_PREFERENCES (except where they have differing structure)? If not I wonder if they should be merged and renamed to something common?

Reply:
Perhaps the addition of some EDID or DisplayID properties in the future? Those might not be in the SDP.
My motivation was the IS-04/05/11 integration tests need different values and they can not be reconfigured mid test.